agent/agent.go

```diff
@@ -308,6 +309,21 @@ func handle(conn io.ReadWriter, op byte) error {
 			return err
 		}
 		_, _ = fmt.Fprintf(conn, "New GC percent set to %v. Previous value was %v.\n", perc, debug.SetGCPercent(int(perc)))
+
+	case SetLogLevel:
+		param, err := bufio.NewReader(conn).ReadByte()
+		if err != nil {
+			return err
+		}
+		if param < byte(logrus.PanicLevel) || param > byte(logrus.TraceLevel) {
+			return errors.Errorf("invalid log level %v", param)
+		}
+
+		oldLevel := logrus.GetLevel()
+		newLevel := logrus.Level(param)
+		logrus.SetLevel(newLevel)
+
+		_, _ = fmt.Fprintf(conn, "Log level set to %v. Previous value was %v.\n", newLevel, oldLevel)
```

Comment:
can we just 'log it' instead of fprintf? in the ZDEW i just log this at info and if the user sets it higher, it doesn't log out (like if they set it to warn/error/panic)

Reply:
That printf sending the string back to the client. It's not going to stdout.

Reply:
aaah. missed that. well then it might be useful to log it here too. thx for that clarity :)